shared/logger.ts

```diff
@@ -0,0 +1,62 @@
+import winston from 'winston';
+
+const levels = {
+  error: 0,
+  warn: 1,
+  info: 2,
+  debug: 3,
+};
+
+const colors = {
+  error: 'red',
+  warn: 'yellow',
+  info: 'green',
+  debug: 'blue',
+};
+
+winston.addColors(colors);
+
+const logger = winston.createLogger({
+  levels,
+  format: winston.format.combine(
+    winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
+    winston.format.errors({ stack: true }),
+    winston.format.json()
+  ),
+  defaultMeta: { service: 'ahm' },
+  transports: [
+    // Write all logs with importance level of `error` or less to `error.log`
+    new winston.transports.File({ 
+      filename: 'logs/error.log', 
+      level: 'error',
+      format: winston.format.combine(
+        winston.format.timestamp(),
+        winston.format.errors({ stack: true }),
+        winston.format.json()
+      )
+    }),
+    // Write all logs with importance level of `info` or less to `combined.log`
+    new winston.transports.File({ 
+      filename: 'logs/combined.log',
+      format: winston.format.combine(
+        winston.format.timestamp(),
+        winston.format.json()
+      )
+    }),
+  ],
+});
+
+// in case we want to log to the console, instead of the file, just set the .env variable
+if (process.env.TS_LOG_CONSOLE ===  'true') {
```

Comment:
I am okay to keep this, but I think the more important configuration is `LOG_LEVEL`, allowing you to control the level. This is often an environment variable. 

About pushing logs to file. In case you are following my advice, I meant it around the logs of the inner processes (ZB), not the logs emitted from TS.

> put the internal logs of inner processes in files to keep the terminal output clean.

Reply:
@x3c41a I think @kianenigma means we need to redirect the stdout/stderr of the child process (for zombie-bite) to a file so the logs are not mixed in the terminal. The other logs file `zombie-bite` use are relative to the base directory and will be printed in `stdout`.  I can add the code to handling this inner process logs.
Thx!